src/about-us/index.html

```diff
@@ -117,9 +117,9 @@ <h3 class="h2 bureau-bio_name">Steve Antonakes</h3>
     <div class="content-l">
         <div class="content-l_col-1-2">
             <section class="block
-                        block__border-top
-                        block__padded-top
-                        block__flush-bottom">
+                            block__border-top
+                            block__padded-top
+                            block__flush-bottom">
                 <h2>Engage in our Work</h2>
```

Comment:
I thought we used AP rules https://www.bkacontent.com/how-to-correctly-use-apa-style-title-case/